docs/formats.rst

```diff
@@ -101,31 +101,31 @@ Below are listed capabilities of all supported formats.
 
 .. [#m] See :ref:`bimono`
 .. [#p] Plurals are necessary to properly localize strings with variable count.
-.. [#n] Comments can be used to pass additional information about string to translate.
-.. [#c] Context is used to differentiate same strings used in different scope (for example `Sun` can be used as abbreviated name of day or as a name of our closest star).
-.. [#l] Location of string in source code might help skilled translators to figure out how the string is used.
-.. [#a] Additional states supported by the file format in addition to not translated and translated.
-.. [#x] XML comment placed before the ``<string>`` element is parsed as a developer comment.
+.. [#n] Comments can be used to pass additional info about string to translate.
+.. [#c] Context is used to differentiate identical strings used in different scopes (for example `Sun` can be used as an abbreviated name of the day "Sunday" or as the name of our closest star).
+.. [#l] Location of a string in source code might help proficient translators figure out how the string is used.
+.. [#a] Additional states supported by the file format in addition to "Not translated" and "Translated".
+.. [#x] XML comment placed before the ``<string>`` element, parsed as a developer comment.
 .. [#f] See :ref:`custom-checks`
-.. [#po] The Gettext type comments are used as flags.
-.. [#xl] The flags are extracted from non standard attibute ``weblate-flags`` for all XML based formats. Additionally ``max-length:N`` is supported through ``maxwidth`` attribute as defined in the Xliff standard, see :ref:`xliff-flags`.
+.. [#po] The gettext type comments are used as flags.
+.. [#xl] The flags are extracted from the non-standard attibute ``weblate-flags`` for all XML based formats. Additionally ``max-length:N`` is supported through the ``maxwidth`` attribute as defined in the XLIFF standard, see :ref:`xliff-flags`.
```

Comment:
attributes?

Reply:
Yes, XML attributes, see http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#maxwidth for specs.